src/helm/benchmark/scenarios/vision_language/utils_latex.py

```diff
@@ -0,0 +1,85 @@
+from typing import Optional, Tuple
+
+import io
+import os
+
+from helm.common.optional_dependencies import OptionalDependencyNotInstalled
+
+try:
+    from latex import build_pdf
+    from pdf2image import convert_from_bytes
+    from PIL import ImageOps
+except ModuleNotFoundError as e:
+    raise OptionalDependencyNotInstalled(
+        f"Optional dependency {e.name} is not installed. "
+        "Please run `pip install latex pdf2image Pillow` to install it."
+    ) from e
+
+TEX_BEGIN = r"""
+\documentclass{article}
+\usepackage{amsmath,amssymb,amsfonts}
+\usepackage{graphicx}
+\usepackage{graphicx}
+\usepackage{amsmath}
+\usepackage{xcolor}
+\usepackage{algorithm}
+\usepackage{algpseudocode}
+\usepackage{stfloats}
+\usepackage{epstopdf}
+\usepackage{pgfplots}
```

Comment:
Are these packages built-in? Or do you have to install them manually? If the latter, it would be good to document this somewhere.

Reply:
I added a comment explaining how to add them